examples/e2e/docker-compose.yml

```diff
@@ -46,6 +46,7 @@ services:
     entrypoint: "/usr/local/bin/worker"
     environment:
       OSSMALWARE_WORKER_SUBSCRIPTION: kafka://worker?topic=workers
+      OSSMALWARE_NOTIFICATION_TOPIC: kafka://notifications
```

Comment:
Should we perhaps be trying to standardise the prefix here (e.g. to `OSSF_MALWARE`)? @oliverchang

Reply:
good point. Let's not make this worse and follow "OSSF_MALWARE"